bin/request-vsp-certs

```diff
@@ -90,6 +117,13 @@ function store_key_and_csr_and_challenge_phrase {
       --file "${FILE_PREFIX}.key"
   fi
 
+  if ! [ "$KEY_EXISTED" ] || ! [ "$BASE64_KEY_EXISTED" ]; then
```

Comment:
Probably worth writing down why we made this decision somewhere (in the commit message?). "If we generated the key, we always want a new version of the base64 encoded version, regardless of whether we already had a base64 encoded key or not."